ComputerVision/SDK/AnalyzeImage/Program.cs

```diff
@@ -34,26 +31,11 @@ static void Main(string[] args)
                 new ApiKeyServiceClientCredentials(subscriptionKey),
                 new System.Net.Http.DelegatingHandler[] { });
             
-            // You must use the same region as you used to get your subscription
-            // keys. For example, if you got your subscription keys from westus,
-            // replace "westcentralus" with "westus".
-            //
-            // Free trial subscription keys are generated in the westcentralus
-            // region. If you use a free trial subscription key, you shouldn't
-            // need to change the region.
-
-            // Specify the Azure region
+            // You must use the same region as you used to get your subscription keys. For example, if you got your subscription keys from westus, replace "westcentralus" with "westus". Free trial subscription keys are generated in the westcentralus region. 
             computerVision.Endpoint = "https://westus.api.cognitive.microsoft.com";
 
-
-
             // localImagePath = @"C:\Documents\LocalImage.jpg"
-            string localImagePath = Directory.GetCurrentDirectory() + @"../../../../../../Images\sample5.png";
-            string[] allfiles = Directory.GetFiles(localImagePath);
-            foreach (var filename in allfiles)
-            {
-                Console.WriteLine(filename);
-            }
+            string localImagePath = Directory.GetCurrentDirectory() + @"../../../../../../Images\sample5.png"; 
```

Comment:
Let's add the relevant images to each project direclty as assets to each project, so they get copied to the ouput folder and the dependency is more obvious.

- In each of the the CS projects, create an Images folder in the project, then add the relevant images as links (right click on the project Images folder in VS -> Add Existing -> find the image and under the Add button drop down select Add as Link)
- On the properties for those images, set Build action to Content and make it copy to the output directory if newer.

Now those images will be copied to the output folder for each console app and you can simply set the localImagePath to something like Images\sample5.png.